packages/chaire-lib-common/src/config/defaultPreferences.config.ts

```diff
@@ -604,11 +605,7 @@ const defaultPreferences: PreferencesModel = {
         }
     },
     proj4Projections: {
-        '4326': {
-            srid: 4326,
-            label: 'WGS84 EPSG:4326 (latitude/longitude)',
-            value: '+title=WGS 84 (long/lat) +proj=longlat +ellps=WGS84 +datum=WGS84 +units=degrees'
-        },
+        [String(constants.geographicCoordinateSystem.srid)]: constants.geographicCoordinateSystem,
         '2950': {
```

Comment:
Maybe not for now, but probably all projections definition should in in constants somewhere and not in a defaultPreferences file.

Reply:
I'll add an issue, there's no reason to support only MTM zone 8 and WSG84. There's a lot more geographic coordinates systems. Preferences could reduce the supported subset of an instance maybe, but Transition should support them all.